rust/protocol/src/storage/traits.rs

```diff
@@ -1,38 +1,70 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Traits defining several stores used throughout the Signal Protocol.
```

Comment:
I would add a word about the usage of `async-trait`, including the fact that it's `async_trait(?Send)`. Maybe some example/dummy implementation would be really great here.

Reply:
I started to look into having the store traits require `Send`, just to see how far I could go, would you mind if I open an issue so that we can discuss it in more details there?

Reply:
@gferon That sounds great, thank you!